openreview/venue/venue.py

```diff
@@ -1009,6 +1015,125 @@ def set_SAC_ethics_review_process(self, sac_ethics_flag_duedate=None):
     def open_reviewer_recommendation_stage(self, start_date=None, due_date=None, total_recommendations=7):
         self.invitation_builder.set_reviewer_recommendation_invitation(start_date, due_date, total_recommendations)
 
+    def ithenticate_create_and_upload_submission(self):
+        if not self.iThenticatePlagiarismCheck:
+            raise openreview.OpenReviewException('iThenticatePlagiarismCheck is not enabled for this venue.')
+        
+        self.invitation_builder.set_iThenticate_plagiarism_check_invitation()
+
+        iThenticate_client = openreview.api.iThenticateClient(self.iThenticatePlagiarismCheckApiKey, self.iThenticatePlagiarismCheckApiBaseUrl)
+        
+        submissions = self.get_submissions()
+        for submission in submissions:
```

Comment:
you can use `tqdm` to follow the progress of the submissions.

Reply:
Done